Add getter for total stakers

oraclown opened the pull request:
Closes #34

tkernell commented on contracts/TellorFlex.sol:
```diff
@@ -153,6 +154,9 @@ contract TellorFlex {
             require(token.transferFrom(msg.sender, address(this), _amount));
         }
         _updateStakeAndPayRewards(msg.sender, _staker.stakedBalance + _amount);
+        if (_staker.startDate == 0) {
```
if someone stakes, then completely unstakes, then stakes again with the same address, this logic won't increase the `totalStakers`

tkernell commented:
This is a good start. A few things though:
- We could probably put the logic for updating the number of stakers in the `_updateStakeAndPayRewards` function. This function is called every time someone's stakedBalance is updated, which coincides with updates to the `totalStakers` count
- A mapping is probably needed for tracking whether a staker address was counted by `totalStakers`. Otherwise, it seems the totalStakers count could drift a bit far from the real number of totalStakers.

oraclown commented:
> This is a good start. A few things though:
> 
> * We could probably put the logic for updating the number of stakers in the `_updateStakeAndPayRewards` function. This function is called every time someone's stakedBalance is updated, which coincides with updates to the `totalStakers` count
> * A mapping is probably needed for tracking whether a staker address was counted by `totalStakers`. Otherwise, it seems the totalStakers count could drift a bit far from the real number of totalStakers.

Thanks for the feedback, sire. I added a bool to `StakeInfo` struct to keep track of whether someone's staked and moved the logic for determining that status to `_updateStakeAndPayRewards`. Lmk if it looks decent
